easybuild/easyconfigs/a/AUGUSTUS/AUGUSTUS-3.3-foss-2018a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'MakeCp'
```

Comment:
@Serpens could you explain why you have changed the `easyblock` here? Is it an upstream change?

Reply:
I changed it because I started writing this easyconfig before I noticed the older one and started from scratch. I can change to `ConfigureMake`, but the `configure` step is skipped anyway, so I'm not sure it makes any difference.